tailwind.config.cjs

```diff
@@ -1,6 +1,11 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,js}"],
+  content: [
+    './src/**/*.css',
+    './src/**/*.js',
+    './src/apis/*.js',
+    './src/main/*.js',
```

Comment:
I am sorry but this has not resolved the issue. The warning is still coming up.
Moreover, `"./src/**/*.{html,js}"` already includes paths as `'./src/main/*.js'` and `'./src/apis/*.js'`

- An asterisk (`*`) — matches everything except slashes (path separators), hidden files (names starting with .).
- A double star or globstar (`**`) — matches zero or more directories.

You figured out correctly that the problem lies inside this config file, but your solution is wrong.

[TailwindCSS docs](https://tailwindcss.com/docs/content-configuration#configuring-source-paths) to help.

Tailwind is using `fast-glob` library for matching these patterns. You can find more about its syntax on [its GitHub page](https://github.com/mrmlnc/fast-glob#basic-syntax).

Reply:
So when I set the body's background colour to green, I would expect it to be green. But it is not.
I pulled your branch with your configuration.

![image](https://user-images.githubusercontent.com/65303812/221840323-7861f45e-f0fa-4531-8205-ef5d4792132c.png)

I don't think I can just ignore those warnings.

Reply:
let me see